R/tar-terra.R

```diff
@@ -0,0 +1,54 @@
+#' @export
+tar_terra_rast <- function(name,
+                           command,
+                           pattern = NULL,
+                           packages = targets::tar_option_get("packages"),
+                           tidy_eval = targets::tar_option_get("tidy_eval"),
+                           library = targets::tar_option_get("library"),
+                           format = format_terra_rast(),
```

Comment:
This here is the main difference (sorry, looks like GitHub is registering this as an entirely  new file since I renamed it)

Reply:
Looks good to me!